sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -33,8 +33,8 @@
 from apache_beam.io import filebasedsource
 from apache_beam.io import iobase
 from apache_beam.io import source_test_utils
-from apache_beam.io.avroio import _AvroSink as AvroSink # For testing
-from apache_beam.io.avroio import _AvroSource as AvroSource # For testing
+from apache_beam.io.avroio import _create_avro_sink as AvroSink  # For testing
```

Comment:
Just use _create_avro_source and _create_avro_sink from tests instead of "as Avro..." ? I think that is cleaner.